database/migrations/sqlite/1714995407000000_event_sourcing_light.up.sql

```diff
@@ -1,6 +1,6 @@
 CREATE TABLE IF NOT EXISTS event_sourcing_light -- aka ESL
 (
-    eslId INTEGER PRIMARY KEY,
+    eslId INTEGER PRIMARY KEY autoincrement,
     created TIMESTAMP,
```

Comment:
This allows us to implement the DBDiscoverCurrentEsldID function for sqlite. SQLITE_SEQUENCE table does not exist if there is no autoincrement index